riptide-opentracing/src/main/java/org/zalando/riptide/opentracing/span/HttpRateLimitSpanDecorator.java

```diff
@@ -0,0 +1,45 @@
+package org.zalando.riptide.opentracing.span;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.opentracing.tag.StringTag;
+import io.opentracing.tag.Tag;
+import org.apiguardian.api.API;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+import static org.zalando.riptide.opentracing.span.HttpHeaderSpanDecorator.mapping;
+
+/**
+ * @see <a href="https://ioggstream.github.io/draft-polli-ratelimit-headers/draft-polli-ratelimit-headers.html">Rate-Limit Header Fields for HTTP</a>
+ */
+@API(status = EXPERIMENTAL)
+public final class HttpRateLimitSpanDecorator extends DelegateSpanDecorator {
+
+    private static final Tag<String> LIMIT =
```

Comment:
`http.rate_limit_limit`?
`http.rate_limit.limit`?

Reply:
second seems more appropriate as 'limit' will be part of 'rate_limit'

Reply:
I'm not sure to which level this will be used, but is there possibility to switch to `rate_limit.remaining` and so on?

Reply:
I started an internal discussion about the proper name for this:
https://github.bus.zalan.do/SRE/opentracing/issues/43